web_export_view/static/src/js/web_export_view.js

```diff
@@ -0,0 +1,106 @@
+            var column_index = 0;
+            var column_header_selector;
+            $.each(view.renderer.columns, function () {
+                if (this.tag == 'field' && (this.attrs.widget === undefined || this.attrs.widget != 'handle')) {
+                    // non-fields like `_group` or buttons
+                    export_columns_keys.push(column_index);
+                    column_header_selector = '.o_list_view > thead > tr> th:not([class*="o_list_record_selector"]):eq('+column_index+')';
+                    export_columns_names.push(view.$el.find(column_header_selector)[0].textContent);
+                }
+                column_index ++;
+            });
+            var export_rows = [];
+            $.blockUI();
+            if (children) {
+                // find only rows with data
+                view.$el.find('.o_list_view > tbody > tr.o_data_row:has(.o_list_record_selector input:checkbox:checked)')
+                .each(function () {
+                    var $row = $(this);
+                    var export_row = [];
+                    $.each(export_columns_keys, function () {
+                        var $cell = $row.find('td.o_data_cell:eq('+this+')')
+                        var $cellcheckbox = $cell.find('.o_checkbox input:checkbox');
+                        if ($cellcheckbox.length) {
+                            export_row.push(
+                                $cellcheckbox.is(":checked")
+                                ? _t("True") : _t("False")
+                            );
+                        }
+                        else {
+                            var text = $cell.text().trim();
+                            if ($cell.hasClass("o_list_number")) {
+                                export_row.push(parseFloat(
+                                    text
+                                    // Remove thousands separator
+                                    .split(_t.database.parameters.thousands_sep)
+                                    .join("")
+                                    // Always use a `.` as decimal separator
+                                    .replace(_t.database.parameters.decimal_point, ".")
+                                    // Remove non-numeric characters
+                                    .replace(/[^\d\.-]/g, "")
+                                ));
+                            }
+                            else {
+                                export_row.push(text);
+                            }
+                        }
+                    });
+                    export_rows.push(export_row);
+                });
+            }
+            session.get_file({
+                url: '/web/export/xls_view',
+                data: {data: JSON.stringify({
+                    model: view.modelName,
+                    headers: export_columns_names,
+                    rows: export_rows
+                })},
+                complete: $.unblockUI
```

Comment:
Can you please add error handler ?
`error: c.rpc_error.bind(c),`
'c' will be crash manager.

Reply:
Any fast way of testing the error binding?

Reply:
May b you can raise a warning or traceback from /web/export/xls_view route and it will be handled via  crashmanager => `error: c.rpc_error.bind(c)`